webapp/sources/ldap-inventory/inventory-repository/src/main/scala/com/normation/inventory/ldap/core/InventoryMapper.scala

```diff
@@ -474,32 +474,32 @@ class InventoryMapper(
       case VirtualMachineType(Virtuozzo)     => OC_OC_VM_VIRTUOZZO
       case VirtualMachineType(LXC)           => OC_OC_VM_LXC
       case PhysicalMachineType               => OC_OC_PM
-      case UnknownMachineType                => OC_OC_PM // we didn't had unknown in that time and physical was the default
+      case UnknownMachineType                => OC_OC_MACHINE
```

Comment:
This normalization was necessary to avoid non-idempotence when saving node fact without a machine. 
We would get it, map to "unknown machine", save it physical machine, leading to an `Updated` change event in place of a `Noop`.